### PR TITLE
(MESI-2018) Insert ModeZone in OB1

### PR DESCRIPTION
**Pull request by Ebubekir-Simsek**
Implemented a method that will create arbitrary amount of ModeZone networks as XElement instances and inserting the MZ after a specific Network called 7.3 - Submodule - Machine network.

010,FC300,****Mode-Zone 010****,Logic Mode Zone 010
020,FC301,****Mode-Zone 020****,Logic Mode Zone 020
030,FC302,****Mode-Zone 030****,Logic Mode Zone 030
Name, FC, Title, Comment

### CODE REVIEW

File: Software/OB1ContentHandler.cs [hunk 1]
```diff
@@ -67,9 +67,10 @@ static IReadOnlyList<XElement> CreateModeZones(ProjectRecipe projectRecipe)
             var networks = new List<XElement>();
             var recipe = RecipeHandler.GetRecipe<ModeZonesRecipe>("ModeZones.txt");
 
+
             foreach (var modeZone in recipe.ModeZones)
             {
-                var creator = new ModeZoneNetworkCreator(modeZone,projectRecipe,languages);
+                var creator = new ModeZoneNetworkCreator(modeZone,projectRecipe,languages,modeZone.Title,modeZone.Comment);
```
Comment (seriks31) [inline Software/OB1ContentHandler.cs:73]: Ta bort onödiga parametrar.

File: XmlCreators/NetworkCreators/ModeZoneNetworkCreator.cs [hunk 2]
```diff
@@ -19,11 +19,10 @@ internal class ModeZoneNetworkCreator : NetworkCreator
         ModeZone modeZone;
         ProjectRecipe projectRecipe;
 
-        public ModeZoneNetworkCreator(ModeZone modeZone, ProjectRecipe projectRecipe, LanguageAssociation languages) : base("","",languages)
+        public ModeZoneNetworkCreator(ModeZone modeZone, ProjectRecipe projectRecipe, LanguageAssociation languages, string comment, string title) : base(title, comment, languages)
```
Comment (seriks31) [inline XmlCreators/NetworkCreators/ModeZoneNetworkCreator.cs:22]: Ta bort commnet och title parametrarna.
I anropet till basklassens konstruktor ska du skicka med modeZone.Title och modeZone.Comment.

**Review by seriks31 — CHANGES_REQUESTED** [review]

**Comment by seriks31** [comment]
Vänta på min branch MESI-2021.

**Comment by Ebubekir-Simsek** [comment]
Currently not valid!

**Pull request closed without merging**